packages/components/src/Tabs/Context.ts

```diff
@@ -1,6 +1,11 @@
 import { createContext, useContext } from 'react';
 
 export const TabContext = createContext<{
-  classNames: { tabs: string; tab: string };
+  classNames: {
```

Comment:
do we than need this still? @OsamaAbdellateef

Reply:
@sarahgm 
`
  const classNames = useClassNames({
    component: 'Tabs',
    size,
    variant,
  });
`
`    <TabContext.Provider value={{ classNames }}>
`
instead of using `useClassNames` multiple times

Reply:
@sarahgm 
is it clear ?